trainer/finetune.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2020 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script for Running, Training, and Evaluating E2E Convrec Experiments."""
+
+# SETUP
+print("Installing dependencies...")
```

Comment:
Why do you have a print statement at the very top instead of in main? Do the imports themselves take a long time?

Reply:
It was a line from the notebook, I took it out.